models/discriminators.py

```diff
@@ -125,8 +125,8 @@ def __init__(self, img_size, kernel_size, num_filters, lr, w_init, bn_init):
     def build(self, input_size):
         activation = lambda l: tf.nn.leaky_relu(l, alpha=0.2)
 
-        self.conv_block_1 = ConvBlock(filters=self.d_dim, kernel_size=(4, 4), strides=(2, 2), padding='same',
-                                      w_init=self.w_init, bn_init=self.bn_init, activation=activation)
+        self.conv_1 = Conv2D(filters=self.d_dim, kernel_size=(4, 4), strides=(2, 2), padding='same',
+                             kernel_initializer=self.w_init, use_bias=False, activation=activation)
```

Comment:
does passing a tf.nn activation function work?

Reply:
Yeah good point. No error gets thrown but I just checked the docs and it doesn't say anything about the tf.nn activations. I guess I'm not sure then. Safer to pass in the standard tf.keras one

Reply:
na the keras one throws warnings when we save. Just apply it separately in the forward pass (call function)